ilc/client.js

```diff
@@ -144,10 +144,20 @@ function getCurrentBasePath() {
     return currentPath.basePath;
 }
 
+function getNormalizedLocation() {
+    // we should respect base tag for cached pages
+    const base = document.querySelector('base');
```

Comment:
Use it only during init & remove after first use while showing warning in console.